src/Middleware/Diagnostics.Abstractions/src/IDeveloperPageExceptionFilter.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Diagnostics
+{
+    /// <summary>
+    /// Provides an extensiblity point for changing the behavior of the DeveloperExceptionPageMiddleware />.
+    /// </summary>
+    public interface IDeveloperPageExceptionFilter
+    {
+        /// <summary>
+        /// An exception handling method.
```

Comment:
When you do get around to docs, it would be good to explain what I'm supposed to do in this method. Should I always call `next`, or only sometimes?